src/zope/app/wsgi/testing.py

```diff
@@ -37,8 +37,8 @@ def __call__(self):
         fn = tempfile.mktemp()
```

Comment:
Bad test is not cleaning after itself!  And creates completely anonymous temporary files.